### PR TITLE
Add EDNS upstream version in the release notes

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.12

Issue: None

Link to docs preview: https://52775--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-nw-external-dns-operator

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -168,6 +168,8 @@ For more information, see xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k
 
 In {product-title} 4.12, the External DNS Operator modifies the format of the ExternalDNS wildcard TXT records on AzureDNS. The External DNS Operator replaces the asterisk with `any` in ExternalDNS wildcard TXT records. You must avoid the ExternalDNS wildcard A and CNAME records having `any` leftmost subdomain because this might cause a conflict.
 
+The upstream version of `ExternalDNS` for an OpenShift Container Platform 4.11 is v0.13.0.
```
Comment (alebedev87) [inline release_notes/ocp-4-12-release-notes.adoc:171]:
```suggestion
The upstream version of `ExternalDNS` for an OpenShift Container Platform 4.12 is v0.13.1. 
```

Comment (xenolinux) [reply release_notes/ocp-4-12-release-notes.adoc]: Addressed

**Review by kcarmichael08 — COMMENTED** [review]
Couple of suggestions; otherwise LGTM

**Pull request merged**